dataland-frontend/src/components/resources/frameworkDataSearch/p2p/P2pDataModel.ts

```diff
@@ -428,8 +428,8 @@ export const p2pDataModel = [
           {
             name: "productionSiteEnergyConsumption",
             label: "Production site energy consumption",
-            description: "Absolute total energy consumption of the company's production sites (MWh)",
-            unit: "",
+            description: "Absolute total energy consumption of the company's production sites",
+            unit: "MWh",
```

Comment:
Again, Unit now no longer visible in upload page. Only acceptable if this will be resolved as part of DALA-2373.

Reply:
To show units here the used component  has to be adapted. Currently only DataPointsFormField and YesNoFormFields show the units based on the DataModel.ts . The NumberFormField does not show units yet. So there are two options: 1) Extend the NumberFormField component to show units. 2) Change the component used to DataPointsFormField.